src/ngraph/frontend/onnx_import/ops_bridge.cpp

```diff
@@ -86,42 +86,194 @@ namespace ngraph
         {
             namespace error
             {
-                struct unknown_operation : ngraph_error
+                struct UnknownOperator : ngraph_error
                 {
-                    explicit unknown_operation(const std::string& op_type)
-                        : ngraph_error{"unknown operation: " + op_type}
+                    explicit UnknownOperator(const std::string& op_type)
```

Comment:
Please add `<string>` header.

Reply:
It is already there, check the top of the file.

Reply:
I don't see it there. It is included in `ops_bridge.hpp` where it is unnecessary.

Reply:
What version of changes are you looking at? None of the commits add/remove the <string> from the file preamble, and it is there.